packages/server/src/controllers/driver/driver-controller.ts

```diff
@@ -11,7 +11,7 @@ export const getDrivers = async (req: Request, res: Response) => {
 	}
 };
 export const postDriver = async (req: Request, res: Response) => {
-	const { name, lat, lng, phone, carModel, licensePlate, date, locationName, locationId, userId } = req.body;
+	const { name, phone, carModel, licensePlate, locationId, userId } = req.body;
```

Comment:
Why did you remove locationName?

Reply:
no locationName sent from FE, it's not required also, because by locaionId we can get all inforamtion

Reply:
Oh okay nevermind, just realised it's postDriver